functions/controller/Qa_QuestionDelete.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+
+class Qa_QuestionDelete
+{
+    private $db;
+
+    public function __construct()
+    {
+        require_once '../functions/datalayer/database.class.php';
+        $database = new Database();
+        $this->db = $database->getConnection();
+    }
+
+    function DeleteQaQuestion($Qid){
+        //variabele (n) uit de url halen
+        $query = "UPDATE question SET questionStatus = 'deleted' WHERE questionName = '$Qid'";
+
+
+        $stm = $this->db->prepare($query);
+        if ($stm->execute()) {
+            Header("Location:Qa_QD.php");
+        } else echo "oeps";
```

Comment:
de else heeft geen open en close brackets { }

Reply:
In dit geval hoeft dat ook niet..

Reply:
hij werkt gewoon maat?